share/github-backup-utils/ghe-gc-disable

```diff
@@ -9,8 +9,8 @@
 set -e
 
 # Bring in the backup configuration
-cd $(dirname "$0")/../..
-. share/github-backup-utils/ghe-backup-config
+export GHE_BACKUP_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../.." && pwd )"
```

Comment:
@snh ah, gotcha. Then leaning towards reverting the change for now and keep using `pwd` for now and address this in a future PR.